src/Germany/Mews.Fiscalizations.Germany.Tests/V2/TssTests.cs

```diff
@@ -41,6 +41,16 @@ public async Task GetTssSucceeds()
             AssertTss(tss.IsSuccess, tss.SuccessResult.Id);
         }
 
+        [Test]
+        public async Task GetAllTSSsSucceeds()
+        {
+            var client = TestFixture.GetFiskalyClient();
```

Comment:
Can we create a TSS here and assert it is in the result?
- Create one
- assert Get
- uninitialize